app/src/main/java/com/appdev/eateryblueandroid/ui/components/core/search/RecentSearchList.kt

```diff
@@ -0,0 +1,74 @@
+package com.appdev.eateryblueandroid.ui.components.core.search
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Icon
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.appdev.eateryblueandroid.R
+import com.appdev.eateryblueandroid.models.Eatery
+import com.appdev.eateryblueandroid.ui.components.core.Text
+import com.appdev.eateryblueandroid.ui.components.core.TextStyle
+import com.appdev.eateryblueandroid.util.recentSearches
+
+@Composable
+fun RecentSearchList(
+    eateries: List<Eatery>,
+    selectEatery: (eatery: Eatery) -> Unit,
+){
+    var recentlySearchEateries = mutableListOf<Eatery>();
+
+    eateries.forEach{
+
+        if(recentSearches.contains(it.id)){
+            recentlySearchEateries.add(it)
+        }
+    }
+
+    Column(
+        verticalArrangement = Arrangement.spacedBy(16.dp),
+        modifier = Modifier.padding(start = 16.dp),
+    ){
+        recentlySearchEateries.forEach{
+            RecentSearchItem(it,selectEatery)
+        }
+    }
+}
+@Composable
+fun RecentSearchItem(
+    eatery: Eatery,
+    selectEatery: (eatery: Eatery) -> Unit = {}
+) {
+    Surface(modifier = Modifier.clickable {
+        selectEatery(eatery)
+    }) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+//     modifier = Modifier.padding(start = 16.dp)
```

Comment:
Not touching this file before it's properly cleaned up. Line 58, commented code. Line 67, empty. Line 68, improper indentation.